doc/classes/CPUParticles2D.xml

```diff
@@ -68,132 +93,194 @@
+			Particles are drawn in order of remaining lifetime.
+		</constant
 		<constant name="PARAM_INITIAL_LINEAR_VELOCITY" value="0" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set initial velocity properties.
 		</constant>
 		<constant name="PARAM_ANGULAR_VELOCITY" value="1" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set angular velocity properties.
 		</constant>
 		<constant name="PARAM_ORBIT_VELOCITY" value="2" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set orbital_velocity properties.
 		</constant>
 		<constant name="PARAM_LINEAR_ACCEL" value="3" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set linear acceleration properties.
 		</constant>
 		<constant name="PARAM_RADIAL_ACCEL" value="4" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set radial acceleration properties.
 		</constant>
 		<constant name="PARAM_TANGENTIAL_ACCEL" value="5" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set tangential acceleration properties.
 		</constant>
 		<constant name="PARAM_DAMPING" value="6" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set damping properties.
 		</constant>
 		<constant name="PARAM_ANGLE" value="7" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set angle properties.
 		</constant>
 		<constant name="PARAM_SCALE" value="8" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set scale properties.
 		</constant>
 		<constant name="PARAM_HUE_VARIATION" value="9" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set hue_variation properties.
 		</constant>
 		<constant name="PARAM_ANIM_SPEED" value="10" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set animation speed properties.
 		</constant>
 		<constant name="PARAM_ANIM_OFFSET" value="11" enum="Parameter">
+			Use with [method set_param], [method set_param_randomness], and [method set_param_texture] to set animation offset properties.
 		</constant>
 		<constant name="PARAM_MAX" value="12" enum="Parameter">
+			Represents the size of the [enum Parameter] enum.
 		</constant>
 		<constant name="FLAG_ALIGN_Y_TO_VELOCITY" value="0" enum="Flags">
+			Use with [method set_flag] to set [member flag_align_y].
 		</constant>
 		<constant name="FLAG_ROTATE_Y" value="1" enum="Flags">
+			Use with [method set_flag] to set [member flag_rotate_y].
```

Comment:
In `CPUParticles2D`, `FLAG_ROTATE_Y` and `FLAG_DISABLE_Z` actually don't do anything. They are only exposed for consistency the `Flags` enums in `CPUParticles` and `Particles{,2D}`, so this would be worth mentioning here.